libethereum/State.cpp

```diff
@@ -44,40 +44,44 @@ const char* StateChat::name() { return EthViolet "⚙" EthWhite " ◌"; }
 
 namespace
 {
-
 /// @returns true when normally halted; false when exceptionally halted.
-bool executeTransaction(Executive& _e, Transaction const& _t, OnOpFunc const& _onOp)
+bool executeTransaction(Executive& _e,
+						Transaction const& _t,
```

Comment:
Hm, why did it do this, looks weird

Reply:
Probably `AlignAfterOpenBracket: Align` affects it, and `TabWidth:        4` while github uses  8

Reply:
I think I'd prefer `AlignAfterOpenBracket: DontAlign` and probably we should set `TabWidth` to 8 to make it look more normal on github

Reply:
Here `AlwaysBreak` matches our current style the most.

GitHub, code reviews, code copy & paste are reasons I'm not a fun of tabs. Tabs seem to work in your own editor only. Do you know anyone using tabs of size 2 in cpp-ethereum & solidity.